bibs/Lauterbach_Hessen.json

```diff
@@ -11,6 +11,20 @@
     "country": "Deutschland",
     "data": {
         "baseurl": "https://opac.lauterbach-hessen.de",
+        "mediatypes": {
+            "0": "ALL",
+            "1": "BOOK",
+            "11": "DVD",
+            "13": "PACKAGE",
+            "19": "BOOK",
+            "2": "MAGAZINE",
+            "26": "GAME_CONSOLE",
+            "3": "CD_SOFTWARE",
+            "30": "EBOOK",
+            "35": "AUDIOBOOK",
+            "6": "CD_MUSIC",
+            "9": "BOARDGAME"
+        },
```

Comment:
mediatypes look wrong (see longer comment above), at least `ALL` needs to be removed.

Reply:
See my [comment](https://github.com/opacapp/opacapp-config-files/pull/11#discussion_r430272819) at Essen, except that here different image names, like `../image/book.gif.S`, are used, but they are not numbers.

Reply:
see reply above.